WidokRaportu

StrongMen120 opened the pull request:
Poprawiona czytelność kodu
Dodana ulepszona lista przedmiotów
Poprawa przycisków
Dodana classa do zmniany colory listy po kliknięciu można wybrać kolor jaki chcemy

StrongMen120 committed:
WidokRaportu

Poprawiona czytelność kodu
Dodana ulepszona lista przedmiotów
Poprawa przycisków
Dodana classa do zmniany colory listy po kliknięciu można wybrać kolor jaki chcemy

StrongMen120 committed:
Poprawki

Poprawkone nazwy
dodana klasa item

MikolajczukKamil commented:
- zaznaczanie fajne ale przyznam się nw jak ono działa :D, raz można zaznaczyć kilka raz tylko 1

![a1](https://user-images.githubusercontent.com/25321219/79233224-d09efb80-7e68-11ea-85e4-8a19e8b5c3ec.jpg)

- czytelność nadal można wypolerować :D 

![tak](https://user-images.githubusercontent.com/25321219/79233228-d1d02880-7e68-11ea-9929-05c9d1ddad46.PNG)

- nw czy id miało być takie jakie jest, dziwnie to wygląda np "(id:123", lepiej by wyglądało "(id: 123)"

StrongMen120 committed:
Poprawka

Małe poprawki

StrongMen120 commented:
Poprawiłem ale nie wiem czemu ja mogę u siebie na telefonie zaznaczyć tylko 1 przedmiot

MikolajczukKamil commented:
> Poprawiłem ale nie wiem czemu ja mogę u siebie na telefonie zaznaczyć tylko 1 przedmiot

spróbuj zrobić tak, zaznacz 1 element scroll aby zniknął i zaznacz coś innego to 1 powinno nadal być zaznaczone

MikolajczukKamil commented:
@Jerry-BloodBerry Jako taki szczegół w zasadzie zauważyłem że przyciski mają białe tło co podczas focusu widać, fajnie by było zrobić ikonki z przezroczystymi tłami i najlepiej jednolite dla każdego widoku takie same w chwili obecnej jak patrzę mamy 3 ikony ok znacząco odbiegające od siebie stylem

Tu przykład kosza z tłem i ok w którym usunąłem tło

![a2](https://user-images.githubusercontent.com/25321219/79240443-1613f680-7e72-11ea-9797-266125990bf5.jpg)

MikolajczukKamil commented:
@StrongMen120 A i oczywiście konflikty są trzeba je rozwiązać aby było można merga zrobić

StrongMen120 commented:
O jakie konflikty chodzi a na rozwiązanie tego problemu z listą na chwilę obecną nie mogę znaleźć rozwiązania 
PS:Mam zmieniać te obrazki na bez tła ?

Jerry-BloodBerry committed:
Kosmetyczne poprawki i porządki

Uporządkowałem trochę napisany przez Ciebie kod. Myślę, że poprawiło to czytelność, a i w projekcie jest trochę większy porządek. Daj znać co sądzisz o tych zmianach.

MikolajczukKamil commented:
![te](https://user-images.githubusercontent.com/25321219/79253295-41531180-7e83-11ea-8898-024d1f187f58.PNG)
te konflikty
te ikony to możne na razie zostawmy, zrobi się zadanie z ikonami i jak będzie śmigać to się raz zrobi dla wszystkich widoków

MikolajczukKamil reviewed (approved):
Konflikty trzeba rozwiązać koniecznie, standardowo zachęcam do dbania o styl (formatowanie) a tak to ok

StrongMen120 committed:
Merge pull request #83 from MikolajczukKamil/widok-raportu-poprawki

Kosmetyczne poprawki i porządki

Jerry-BloodBerry reviewed (approved):
Zgadzam się, że ikonki na przyciskach trzeba zmienić na takie z przezroczystym tłem, ale poza tym wszystko git, można rozwiązać konflikty i mergować 👍

StrongMen120 committed:
WidokRaportu

Poprawione Obrazki
Zmieniona nazwa klasy

Jerry-BloodBerry committed:
Fixed merge conflicts
